docs/RELEASE_NOTES.md

```diff
@@ -82,8 +82,7 @@ This section describes the features of the Ansible Modules for Dell PowerScale f
 
 The Ansible Modules for Dell PowerScale release 3.4.0 supports the following features:
```

Comment:
I can see release version is 3.4.0 here, do we need to update this too ?

Reply:
done